src/routes/exercise-finished/__tests__/ExerciseFinishedScreen.spec.tsx

```diff
@@ -55,6 +56,28 @@ describe('ExerciseFinishedScreen', () => {
     })
   })
 
+  it.skip('should render repetition modal if a module is finished', () => {
```

Comment:
Is the skipping of the test on purpose?

Reply:
@LeandraH yes, it is on purpose because of the devMode. It causes error because on devMode the modal is not appearing.